lib/chibi/repl.scm

```diff
@@ -410,6 +453,8 @@
      (lambda (n) (thread-interrupt! thread))
      (lambda ()
        (protect (exn
+                 ((condition? exn)
```

Comment:
Please remove this.  I just pushed a change allowing customization via the new `repl-print` generic.  If you want, loading `(srfi 35)` can automate this customization with:

```
(define-method (repl-print (c condition?) (out output-port?))
  (repl-print-condition c out))
```

or otherwise provide a convenience procedure to register the method.

Reply:
Oops, added repl-print-exception as well.